rs/nns/governance/src/neuron/types.rs

```diff
@@ -563,9 +556,15 @@ impl Neuron {
     ///
     /// If the neuron refreshed recently, no followee neuron IDs are removed
     /// (and returns 0).
-    pub(crate) fn prune_following(&mut self, now_seconds: u64) -> u64 {
-        let is_fresh =
-            self.voting_power_refreshed_timestamp_seconds >= now_seconds - 7 * ONE_MONTH_SECONDS;
+    pub(crate) fn prune_following(
+        &mut self,
+        voting_power_economics: &VotingPowerEconomics,
+        now_seconds: u64,
+    ) -> u64 {
+        let is_fresh = self.voting_power_refreshed_timestamp_seconds
+            >= now_seconds
+                - voting_power_economics.get_start_reducing_voting_power_after_seconds()
+                - voting_power_economics.get_clear_following_after_seconds();
```

Comment:
And this is where VotingPowerEconomics is used to determine whether to delete following from a neuron, replacing some hard-coded constants.